rtgui/history.cc

```diff
@@ -155,8 +155,10 @@ History::History (bool bookmarkSupport) : blistener(NULL), tpc (NULL), bmnum (1)
 void History::initHistory ()
 {
 
+    selchangehist.block(true);
```

Comment:
The `ConnectionBlocker` helper class is available on `gtk3` and should be used here IMHO.